packages/web-components/fast-element/src/components/element-controller.ts

```diff
@@ -461,12 +470,24 @@ export class ElementController<TElement extends HTMLElement = HTMLElement>
         if (definition === void 0) {
             throw FAST.error(Message.missingElementDefinition);
         }
+        if (elementControllerStrategy === undefined) {
```

Comment:
Instead of doing this should we just set the strategy right after the close of the class definition? The we don't have to do the undefined check every time. Similar to the way we are handling the style strategy I think.

Reply:
sure, yeah that makes sense